userspace/libsinsp/user_event.cpp

```diff
@@ -19,6 +19,7 @@ along with sysdig.  If not, see <http://www.gnu.org/licenses/>.
 #include "sinsp.h"
 #include "sinsp_int.h"
 #include "user_event.h"
+#include <regex>
```

Comment:
my centos6 machine got messed up, but yes it seems that they released it in 4.8 but it was [buggy](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53631). I'll replace it with posix regex